python/lsst/obs/hsc/makeHscRawVisitInfo.py

```diff
@@ -0,0 +1,73 @@
+#
+# LSST Data Management System
+# Copyright 2016 LSST Corporation.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import astropy.units
+import astropy.time
+
+from lsst.afw.geom import degrees
+from lsst.afw.coord import Coord, IcrsCoord, Observatory, Weather
+from lsst.daf.butlerUtils import MakeRawVisitInfo
+
+__all__ = ["MakeHscRawVisitInfo"]
+
+
+class MakeHscRawVisitInfo(MakeRawVisitInfo):
+    """Make a VisitInfo from the FITS header of a Subaru HSC image
+
+    Boresight rotation angle is apparently not available.
```

Comment:
There are no comments on the HSC engineering data I looked at.  Here's what I have from a Suprime-Cam frame (`SUPA01270732.fits`):

    INR-STR =               65.186 / [degree] Inst. rotator angle at start          
    INR-END =               65.185 / [degree] Inst. rotator angle at end            
    INST-PA =               90.000 / Position Angle of Instrument                   

Oooh!  A more recent HSC exposure (`HSCA08609518.fits`) has comments on the headers too:

    INR-STR =              -26.982 / [degree] Inst. rotator angle at start          
    INR-END =              -26.982 / [degree] Inst. rotator angle at end            
    INST-PA =                0.000 / [degree] Position Angle of the Instrument Flang